client/js/table-view.js

```diff
@@ -74,10 +76,50 @@ class TableView{
         this.formulaBarEl.addEventListener('keyup', this.handleFormulaBarChange.bind(this));
     }
 
+    renderSum(collie, total) {
+        const whichColumn = document.querySelector(`#sum-${collie}`);
```

Comment:
I think you have a very good point. I didn't even consider a different approach other than assigning each foot cell a unique number upon the footer rendering to be honest; I did not know that was a brittle approach though. 

However, I am using the input bar listener to call that renderSum upon every keyup. If the table was quite large would it be expensive to allocate the new array and stuff all TFOOT TD's in there on each call? But, would I then instead use a closure to initialize the array once and return a function searching that array? But, what if I wanted to dynamically add columns in a new feature? 

You have me thinking and I appreciate it. It seems like a bit of a rabbit hole for me. I think I might make a big mess if I try to implement it in this specific project. I will attempt it though! And, I will also be thinking about the strategy you suggested in future ones though :)